### DISCUSSION TITLE
Apollo + Sentry + Next.js

### DISCUSSION THREAD

**Discussion by mikevercoelen** [vercel/next.js#24095] [Help]
I'm struggling with the Sentry integration of Next.js and Apollo.

I've taken my boilerplate from: 

Now the next step I want to do, is make sure Apollo reports GraphQL endpoint errors to Sentry. I found this: https://www.npmjs.com/package/apollo-link-sentry

They are calling `Sentry.init` from `@sentry/react` but the boilerplate doesn't use that at all.

ANOTHER QUESTION (similar):

I have a hook, where I initialize my session called `useSession` in this session I want to add Sentry context (to set the user) and call `captureException` etc.

However, Sentry is referring to `import * as Sentry from '@sentry/react'` which is not what's being used in the boilerplate.

How can I set this up correctly? Tips / tricks / recommendations?

**Comment by mikevercoelen** [comment]
Bump

**Comment by stigkj** [comment]
Which boilerplate is it? Seems it is left out.
